### ISSUE TITLE
Raft overview link in documentation is broken

### ISSUE THREAD

**Issue by jamshidi799** [etcd-io/website#815]
Hi. In [learning/persistent-storage-files](https://etcd.io/docs/v3.5/learning/persistent-storage-files/) page of doc on the `Prerequisites` section, the Raft overview link gets 404

**Comment by jmhbnz** [comment]
Hey @jamshidi799 - Thanks for letting us know about this. If you have capacity please raise a pull request to fix this for documentation versions `v3.5` and `v3.6`.

The correct link we should use is https://raft.github.io/raft.pdf.

These are the two files that would need to be updated.

```bash
content/en/docs/v3.6/learning/persistent-storage-files.md
content/en/docs/v3.5/learning/persistent-storage-files.md
```

**Comment by jamshidi799** [comment]
Thank you for providing the correct link. Of course, I will raise a pull request ❤️
